vector/src/main/res/values/colors.xml

```diff
@@ -132,4 +132,20 @@
     <color name="vctr_chat_effect_snow_background_light">@color/black_alpha</color>
     <color name="vctr_chat_effect_snow_background_dark">@android:color/transparent</color>
 
+    <attr name="vctr_voice_message_lock_background" format="color" />
+    <color name="vctr_voice_message_lock_background_light">#FFF3F8FD</color>
+    <color name="vctr_voice_message_lock_background_dark">#22252B</color>
+
+    <attr name="vctr_voice_message_playback_background" format="color" />
+    <color name="vctr_voice_message_playback_background_light">#FFE3E8F0</color>
+    <color name="vctr_voice_message_playback_background_dark">#FF394049</color>
+
+    <attr name="vctr_voice_message_play_pause_button_background" format="color" />
+    <color name="vctr_voice_message_play_pause_button_background_light">@android:color/white</color>
+    <color name="vctr_voice_message_play_pause_button_background_dark">#FF8E99A4</color>
+
+    <attr name="vctr_voice_message_recording_playback_background" format="color" />
+    <color name="vctr_voice_message_recording_playback_background_light">#FFE3E8F0</color>
+    <color name="vctr_voice_message_recording_playback_background_dark">#FF394049</color>
```

Comment:
To check, it should be not necessary to add all those attributes

Reply:
Should be discussed with the design team.

Reply:
ok